modules/exploits/windows/scada/advantech_webaccess_webvrpcs_bof.rb

```diff
@@ -0,0 +1,141 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
```

Comment:
I overlooked this earlier, but per [Exploit Ranking](https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking) please change to `GoodRanking`.